src/app/tests/suites/certification/Test_TC_S_1_1.yaml

```diff
@@ -151,7 +151,7 @@ tests:
       response:
           constraints:
               type: list
-              contains: [0, 1, 2, 3, 4, 5, 6, 40, 41, 42]
+              contains: [0, 1, 2, 3, 4, 5, 6, 64, 65, 66]
```

Comment:
Yes, I could have swore that these commands were all tied together because they all had the same time resolution of a 10th of a second, but after reviewing the spec, it doesn't seem to be the case.

Still, should we at least tie EnhancedAdd and EnhancedView?